src/status_im/subs/communities.cljs

```diff
@@ -107,7 +107,10 @@
                                              pending? (update acc :pending conj community)
                                              :else    (update acc :opened conj community))))
                                        {:joined [] :pending [] :opened []}
-                                       (vals communities))]
+                                       (sort-by (fn [{:keys [last-opened-at joined-at]}]
```

Comment:
This reducer seems to be going out of control. Can you simplify it  ?
```clj
(reduce (fn [acc community]
          (let [joined?      (:joined community)
                community-id (:id community)
                pending?     (boolean (get requests community-id))]
            (cond
              joined?  (update acc :joined conj community)
              pending? (update acc :pending conj community)
              :else    (update acc :opened conj community))))
        {:joined [] :pending [] :opened []}
        (sort-by (fn [{:keys [last-opened-at joined-at]}]
                   (or last-opened-at joined-at))
                 #(compare %2 %1)
                 (vals communities)))
```

I think we can get rid of the reducer completely. Something like this should be doable:

```clj
(->> communities
         vals
         (sort-by ...)
         (group-by ...)
         (merge-with ...))
```
Links to relavent functions:
- https://clojuredocs.org/clojure.core/group-by
- https://clojuredocs.org/clojure.core/merge-with